### PR TITLE
Fix of typos and associated update of conversion script 

### PR DESCRIPTION
**Pull request by CamielCartignij**

### CODE REVIEW

File: ThermofluidStream/Resources/Scripts/Conversion/ConvertTFS_from_1.0.0_to_1.1.0.mos
Comment (CamielCartignij) [inline ThermofluidStream/Resources/Scripts/Conversion/ConvertTFS_from_1.0.0_to_1.1.0.mos:1]: Please check correctness of conversion @tobolar

File: ThermofluidStream/Boundaries/PhaseSeparator.mo [hunk 1]
```diff
@@ -1,5 +1,5 @@
 within ThermofluidStream.Boundaries;
-model PhaseSeparator "Parent to Reciever and Accumulator models"
+model PhaseSeparator "Parent to Receiver and Accumulator models"
```
Comment (nieweber) [inline ThermofluidStream/Boundaries/PhaseSeparator.mo:2]:
```suggestion
model PhaseSeparator "Parent to receiver and accumulator models"
```

File: ThermofluidStream/Boundaries/Reservoir.mo [hunk 2]
```diff
@@ -1,15 +1,15 @@
 within ThermofluidStream.Boundaries;
-model Reservoir "Model of a Resorvoir"
+model Reservoir "Model of a Reservoir"
```
Comment (nieweber) [inline ThermofluidStream/Boundaries/Reservoir.mo:2]:
```suggestion
model Reservoir "Model of a reservoir"
```

File: ThermofluidStream/Boundaries/Reservoir.mo [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
   extends Internal.PartialVolume(final useHeatport = false, final initialize_pressure=false, final p_start=0, final A = 0, final U=0);
 
-  parameter Boolean pEnvFromInput = false "Enable Input";
+  parameter Boolean pEnvFromInput = false "Enable input";
   parameter SI.Area A_surf(displayUnit="cm2")=0.01 "Base area of medium";
-  parameter SI.Pressure p_env_par=1e5 "Environment Pressure"
+  parameter SI.Pressure p_env_par=1e5 "Environment pressure"
```
Comment (nieweber) [inline ThermofluidStream/Boundaries/Reservoir.mo:7]:
```suggestion
  parameter SI.Pressure p_env_par=1e5 "Environmental pressure"
```
In general I think "ambient pressure" would be the better term to use but then we'd have to change the name of the parameter (and conversion script) as well...

File: ThermofluidStream/Undirected/Boundaries/PhaseSeparator.mo [hunk 4]
```diff
@@ -1,5 +1,5 @@
 within ThermofluidStream.Undirected.Boundaries;
-model PhaseSeparator "Parent to Reciever and Accumulator models"
+model PhaseSeparator "Parent to Receiver and Accumulator models"
```
Comment (nieweber) [inline ThermofluidStream/Undirected/Boundaries/PhaseSeparator.mo:2]:
```suggestion
model PhaseSeparator "Parent to receiver and accumulator models"
```

File: ThermofluidStream/Undirected/Boundaries/Reservoir.mo [hunk 5]
```diff
@@ -1,15 +1,15 @@
 within ThermofluidStream.Undirected.Boundaries;
-model Reservoir "Model of a Resorvoir"
+model Reservoir "Model of a Reservoir"
```
Comment (nieweber) [inline ThermofluidStream/Undirected/Boundaries/Reservoir.mo:2]:
```suggestion
model Reservoir "Model of a reservoir"
```

File: ThermofluidStream/Undirected/Boundaries/Reservoir.mo [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
   extends Internal.PartialVolume(final useHeatport=false, final initialize_pressure=false, final A=0, final U=0);
 
-  parameter Boolean pEnvFromInput = false "Enable Input";
+  parameter Boolean pEnvFromInput = false "Enable input";
   parameter SI.Area A_surf(displayUnit="cm2")=0.01 "Base area of medium";
-  parameter SI.Pressure p_env_par=1e5 "Environment Pressure"
+  parameter SI.Pressure p_env_par=1e5 "Environment pressure"
```
Comment (nieweber) [inline ThermofluidStream/Undirected/Boundaries/Reservoir.mo:7]:
```suggestion
  parameter SI.Pressure p_env_par=1e5 "Environmental pressure"
```

**Review by nieweber — CHANGES_REQUESTED** [review]

File: ThermofluidStream/Boundaries/Reservoir.mo (hunk 3 above)
Comment (CamielCartignij) [reply ThermofluidStream/Boundaries/Reservoir.mo:7]: Changing seems like a good idea for consistency. Currently both ambient and environment(al) are used in different places.

Comment (nieweber) [reply ThermofluidStream/Boundaries/Reservoir.mo]: Mhh yes then we should change it to be consistent.

**Review by nieweber — APPROVED** [review]

File: ThermofluidStream/Resources/Scripts/Conversion/ConvertTFS_from_1.0.0_to_1.1.0.mos
Comment (tobolar) [reply ThermofluidStream/Resources/Scripts/Conversion/ConvertTFS_from_1.0.0_to_1.1.0.mos:1]: See 121414581eb9f942b49065f834331ce3b8cc0598

**Pull request merged**